File: app/assets/javascripts/models/conyugue.js
// for more details see: http://emberjs.com/guides/models/defining-models/

Bodega.Conyugue = DS.Model.extend({
  nombre: DS.attr('string'),
  apellido: DS.attr('string'),
  direccion: DS.attr('string'),
  nacionalidad: DS.attr('string'),
  cedula: DS.attr('string'),
  fechaNacimiento: DS.attr('date'),
  lugarNacimiento: DS.attr('string'),
  empleador: DS.attr('string'),
  actividadEmpleador: DS.attr('string'),
  cargo: DS.attr('string'),
  profesion: DS.attr('string'),
  ingresoMensual: DS.attr('number'),
  conceptoOtrosIngresos: DS.attr('string'),
  otrosIngresos: DS.attr('number')
});

File: app/models/conyugue.rb
# == Schema Information
#
# Table name: conyugues
#
#  id                      :integer          not null, primary key
#  nombre                  :string(255)
#  apellido                :string(255)
#  nacionalidad            :string(255)
#  cedula                  :string(255)
#  fecha_nacimiento        :date
#  lugar_nacimiento        :string(255)
#  empleador               :string(255)
#  actividad_empleador     :string(255)
#  cargo                   :string(255)
#  profesion               :string(255)
#  ingreso_mensual         :float
#  concepto_otros_ingresos :string(255)
#  otros_ingresos          :float
#

class Conyugue < ActiveRecord::Base
  scope :by_id, -> value{ where("id = ?", "#{value}") }
end
